packages/x-components/src/wiring/__tests__/debounce-operator.spec.ts

```diff
@@ -0,0 +1,195 @@
+import { createResultStub } from '../../__stubs__/index';
```

Comment:
Because `debounce` and `throttle` operators tests were too long, I decided to extract them into its own files, and make a generic utility `createWire`